File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.PersonService.Models/get-person_result.ts


import { GetPerson_Item } from './get-person_item';
import { IGetPerson_Result } from './iget-person_result';

export class GetPerson_Result implements IGetPerson_Result {
  
	public Item?: GetPerson_Item;
  
	public ErrorMessages: string[] = null;
}

File: GTTS/INL.PersonService.Models/GetPerson_Result.cs
using System;
using System.Collections.Generic;
using System.Text;
using INL.Services.Models;

namespace INL.PersonService.Models
{
    public class GetPerson_Result : BaseResult, IGetPerson_Result
    {
        public GetPerson_Item Item { get; set; }
    }
}
